Benchmarks/Thresholds/5.10/CertificatesBenchmark.Parse_WebPKI_Roots_from_DER.p90.json

```diff
@@ -1,8 +1,8 @@
 {
-  "mallocCountTotal" : 5506,
+  "mallocCountTotal" : 4851,
   "memoryLeaked" : 0,
   "readSyscalls" : 0,
-  "releaseCount" : 120084,
-  "retainCount" : 108506,
+  "releaseCount" : 120271,
+  "retainCount" : 109425,
```

Comment:
Can we remove the retain/release counts from the output as well please?

Reply:
Done.